modules/transports/core/nhttp/pom.xml

```diff
@@ -207,6 +207,16 @@
             <classifier>runtime</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito2</artifactId>
```

Comment:
Have we defined the dependency versions in parent? Just checking since it not done in this PR.

Reply:
Yes. It is already there